src/styles/blocks/calendar.scss

```
@@ -0,0 +1,51 @@
+* {
+  margin: 0;
+  box-sizing: border-box;
```

```suggestion
* {
  box-sizing: border-box;
```
It is better not to reset margins using * selector